x-pack/plugins/elastic_assistant/server/ai_assistant_data_clients/conversations/types.ts

```diff
@@ -76,7 +77,8 @@ export interface CreateMessageSchema {
     };
   }>;
   api_config?: {
-    connector_id?: string;
+    action_type_id: string;
+    connector_id: string;
```

Comment:
Should `connector_id` and now `action_type_id` remain optional? Does this mean we can't create system conversations (like `Event Summary`) without first having a connector configured? Maybe that's fine as conversations are locked to `Welcome` till a connector is present?

Reply:
apiConfig can be undefined, but once it is defined it will require the `connector_id` and `action_type_id`. Does that make sense?